canonical-web-and-design/ubuntu.com
# Update /managed secondary nav

## Done

- Update secondary nav `Overview`, `Kafka`, `Cassandra` to `Overview`, `OpenStack`, `Kubernetes`, `Apps` 
- Add `Kafka`, `Cassandra` to `Apps`' secondary nav
- Redirect `/managed/kafka` to `/managed/apps/kafka`
- Redirect `/managed/Cassandra` to `/managed/apps/Cassandra`
- Fix the 404 error on `/managed/contact-us`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1fLRWW-Gtslaia2ypSrLJVBxZVHo5FY5Q84o97Gs2-w4/edit).

## Issue / Card

Fixes [#4370](https://github.com/canonical-web-and-design/web-squad/issues/4370#event-5199410812)

## Screenshots

![image](https://user-images.githubusercontent.com/57550290/130624562-fad942bf-6aa4-4ac6-86e3-a14d99c7ee16.png)
![image](https://user-images.githubusercontent.com/57550290/130624740-e258a804-5b6d-4ff2-9dfa-1b355e37f69a.png)
